ocs_ci/deployment/aws.py

```diff
@@ -79,6 +79,22 @@ def add_node(self):
         # TODO: Implement later
         super(AWSBase, self).add_node()
 
+    def destroy_volumes(self):
```

Comment:
Hope that once we will use just operator we will not have to deal with the volumes at all.